epictrack-api/src/api/services/work_phase.py

```diff
@@ -109,14 +109,14 @@ def find_multiple_works_phases_status(
         result_dict = {}
         work_ids = list(work_params_dict.keys())
 
-        work_phases_dict, total_work_phases = cls._query_work_phases(work_ids)
+        work_phases_dict = cls._query_work_phases(work_ids)[0]
 
         index = 1
         for work_id, work_phase_id in work_params_dict.items():
             result_dict[work_id] = cls._find_work_phase_status(
                 work_id, work_phase_id, work_phases_dict.get(work_id, [])
             )
-            result_dict["is_last_phase"] = index == total_work_phases
+            # result_dict["is_last_phase"] = index == total_work_phases
             index = index + 1
```

Comment:
I think we can remove `index = index + 1` from here since it has no use inside the loop anymore.